Source/WebCore/platform/graphics/gstreamer/MediaPlayerPrivateGStreamer.cpp

```diff
@@ -3259,6 +3285,49 @@ void MediaPlayerPrivateGStreamer::elementSetupCallback(MediaPlayerPrivateGStream
 #endif // USE(WESTEROS_SINK)
 }
 
+OdhDrm MediaPlayerPrivateGStreamer::AvContextGetterImpl::getDrm()
+{
+    OdhDrm drmType = {OdhDrm::NONE};
```

Comment:
please refactor "if chain" : https://stackoverflow.com/questions/24430504/how-to-avoid-if-chains

Reply:
done